delivery-metrics/sql/create_delivery_metrics_db.sql

```diff
@@ -0,0 +1,105 @@
+# drop tables
+
+DROP TABLE IF EXISTS deliverable;
+DROP TABLE IF EXISTS deliverable_quad_map;
+DROP TABLE IF EXISTS epic;
+DROP TABLE IF EXISTS epic_deliverable_map;
+DROP TABLE IF EXISTS issue;
+DROP TABLE IF EXISTS issue_history;
+DROP TABLE IF EXISTS issue_sprint_map;
+DROP TABLE IF EXISTS sprint;
+DROP TABLE IF EXISTS quad;
+
+# create tables
+
+CREATE TABLE deliverable (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	guid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	pillar TEXT, 
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP 
+);
+
+CREATE TABLE deliverable_quad_map (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	deliverable_id INTEGER NOT NULL,
+	quad_id INTEGER,
+	d_effective DATE NOT NULL,
+	t_modified TIMESTAMP,
+	UNIQUE(deliverable_id, d_effective)
+);
+
+CREATE TABLE epic (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	guid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	t_created TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+	t_modified TIMESTAMP 
+);
+
+CREATE TABLE epic_deliverable_map (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	epic_id INTEGER NOT NULL,
+	deliverable_id INTEGER,
+	d_effective DATE NOT NULL,
+	t_modified TIMESTAMP,
+	UNIQUE(epic_id, d_effective)
+);
+
+CREATE TABLE issue (
+	id INTEGER PRIMARY KEY AUTOINCREMENT,
+	guid TEXT UNIQUE NOT NULL,
+	title TEXT NOT NULL,
+	type TEXT NOT NULL,
```

Comment:
Type can also be NULL unless your setting a default elsewhere in the insert statement

Reply:
In `DeliveryMetricsDataLoader._extractIssue`, a null value for `issue_type` is replaced with `Undefined`:
```
issue_type = item.get('issue_type') or 'Undefined'
```